# Papers discussing / utilizing enhancer - promoter contacts / loops / interactions


## Loop Callers






## Inferring 3D Structure from chromatin contacts

[Pastis-NB: Nelle Varoquaux et al. Bioinformatics 2023](https://pubmed.ncbi.nlm.nih.gov/36594573/) Modeling overdispersion of HIC data to predict 3D structures, by using NB model instead of Poisson model from their previously published Pastis-PM method.




## Structural Variation (CNV, SV) analysis






## CRISPR / experimental validation of E-P contacts, regulatory regions

[CRISPRQTL - A Genome-wide Framework for Mapping Gene Regulation via Cellular Genetic Screens - Gasperini et al. Cell 2019](https://pubmed.ncbi.nlm.nih.gov/30612741/) validated E-P pairs by high throughput CRISPR perturbations in thousands of cells. Tested 1119 candidate enhancers in K562. Enhancers were intergenic DHS - H3K27ac, P300, GATA, and RNA PolII binding sites. 5611 of 12984 genes expressed in K562 fell within 1 Mb of the tested enhancers.

[Bergman et al. Nature 2022](https://pubmed.ncbi.nlm.nih.gov/35594906/) proposed Exp-STARRseq - compatibility rule between enhancer and promoter sequences.

[Nasser et al. Nature 2021](https://pubmed.ncbi.nlm.nih.gov/33828297/) applied ABC score to prioritize regulatory enhancers for target genes, and compared against lots of E-P linking strategies.

[Fulco et al. Nat Genet 2019](https://pubmed.ncbi.nlm.nih.gov/31784727/) proposed ABC score to prioritize regulatory enhancers for target genes.

[Sahu et al. Nat Genet 2022](https://pubmed.ncbi.nlm.nih.gov/35190730/) measured transcriptional activity using MPRA and observed using ML models that TFs act in additive manner and enhancers affect promoters without relying on specific TF-TF interactions. Few TFs are strongly active in a cell, and individual TFs may have multiple regulatory activities.

